internal/cmd/kickstart/kickstart.go

```diff
@@ -0,0 +1,160 @@
+package kickstart
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"pkg.package-operator.run/cardboard/kubeutils/kubemanifests"
+
+	"package-operator.run/internal/packages"
+)
+
+type Kickstarter struct {
+	stdin io.Reader
+}
+
+func NewKickstarter(stdin io.Reader) *Kickstarter {
+	return &Kickstarter{
+		stdin: stdin,
+	}
+}
+
+func (k *Kickstarter) KickStart(
+	ctx context.Context,
+	pkgName string,
+	inputs []string,
+) (msg string, err error) {
+	if err := os.Mkdir(pkgName, os.ModePerm); err != nil {
+		return "", fmt.Errorf("create directory: %w", err)
+	}
+
+	var objects []unstructured.Unstructured
+	for _, input := range inputs {
+		newObjects, err := k.getInput(ctx, input)
+		if err != nil {
+			return "", fmt.Errorf("get input: %w", err)
+		}
+
+		objects = append(objects, newObjects...)
+	}
+
+	rawPkg, res, err := packages.Kickstart(ctx, pkgName, objects)
+	if err != nil {
+		return "", err
+	}
+	for path, data := range rawPkg.Files {
+		path = filepath.Join(pkgName, path)
+		if err := os.MkdirAll(filepath.Dir(path), os.ModePerm); err != nil {
+			return "", fmt.Errorf("creating directory: %w", err)
+		}
+		if err := os.WriteFile(path, data, os.ModePerm); err != nil {
+			return "", fmt.Errorf("writing file: %w", err)
+		}
+	}
+
+	msg = fmt.Sprintf("Kickstarted the %q package with %d objects.", pkgName, res.ObjectCount)
+	report, ok := reportGKsWithoutProbes(res.GroupKindsWithoutProbes)
+	if ok {
+		msg += "\n" + report
+	}
+	return msg, nil
```

Comment:
There will be no report generated if there are no GKsWithoutProbe that require probing.
Because this function is not used anywhere else, I could include the newline by default, but I think it's cleaner to report to the caller and have the newline added here.